jina/types/request/__init__.py

```diff
@@ -24,6 +24,8 @@ class Request(ProtoTypeMixin):
     """
 
     def __getattr__(self, name: str):
+        if name == 'non_empty_fields':
```

Comment:
can u add a test for this?

Reply:
added

Reply:
non_empty_fields was completely removed from Request in https://github.com/jina-ai/jina/pull/4102